src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const reverseRobot = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (reverseRobot[value] !== undefined) {
```

Comment:
I think we have this in checklist
it`s better to use hasOwnProperty insted of  !== undefined